### ISSUE TITLE
New quote also at pages reload.

### ISSUE THREAD

**Issue by lck-git** [cmsimple-xh/quoteoftheday#4]
Extend the configuration with this option. See [CMSimple_XH-Forum](https://cmsimpleforum.com/viewtopic.php?f=4&t=16151&p=75205&sid=390c37007053470ad7710c45d84c24a6#p75204) for more info.

**Comment by cmb69** [comment]
Makes sense. We may also want to support "arbitrary" periods, e.g. an hour, a week, etc.
